Dockerfile

```
@@ -0,0 +1,17 @@
+FROM ubuntu:latest
+#ubuntu:roller
+
+RUN apt update -y; exit 0
+RUN apt-get install build-essential -y
+RUN apt-get install nano cmake git doxygen cmake-curses-gui wget -y
+ARG DEBIAN_FRONTEND=noninteractive #should be change
+RUN apt-get install imagemagick libboost-all-dev cimg-dev -y
+
+RUN git clone https://github.com/ferxxp/fast_methods_2023.git /fast_methods/

```

This is a great contribution, thank you!
Since this will be uploaded to this repo, perhaps we should use the git repo jvgomez/fast_methods ?

---

Thanks.

I'm a little noob with git and github, but yes. 
If you accept the merge I believe it will be added to the repo jvgomez/fast_methods.

---

What I meant is that this dockerfile should clone this repo (otherwise it will clone a fork)
```suggestion
RUN git clone https://github.com/jvgomez/fast_methods.git /fast_methods/

```